src/app/pages/profile/transaction-information/transaction-information.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="transaction__row">
+  <div class="transaction-header">
+    <h3 class="heading-tertiary">Successful transactions</h3>
+  </div>
+  <div class="" *ngIf="isEmpty">
+    <h2>No transcations now</h2>
+  </div>
+  <div class="col-empty" *ngIf="sucessTransactions.length === 0">
```

Comment:
I think it would be better here to use `ngIf; else`. Here's an example of what I mean:

Instead of:
```javascript
<div class="col-empty" *ngIf="sucessTransactions.length === 0">
  ...
</div>

<div class="col-success-transaction" *ngIf="sucessTransactions.length !== 0">
  ...
</div>
```

You can do:
```javascript
<div class="col-empty" *ngIf="!sucessTransactions.length; else notEmpty">
  ...
</div>

<ng-template #notEmpty>
  <div class="col-success-transaction">
    ...
  </div>
</ng-template>
```

Reply:
Thanks @kelvinndmo for your feedback, however, this will not work as expected.

Reply:
Hey @Jaman-dedy , did you encounter an issue while implementing this ?

Reply:
Thanks, @kelvinndmo I worked on that